client/ProfileComponents/Profile.jsx

```diff
@@ -17,21 +16,21 @@ class Profile extends React.Component {
   render() {
     return (
       <div>
-        <div className="pic-position">
-          <Pic photo={this.props.fakePhoto} />
-        </div>
-        <div className="bio-position">
-          <BioList userInfo={this.props.fakeUser} />
-        </div>
-        <div className="posts-position">
-          <PostList posts={this.props.fakePosts} />
-        </div>
-        <div className="friends-position">
-          <Friends friends={this.props.fakeFriends} />
-        </div>
+        <h1>
+          <small>Profile Page for HR Peeps</small>
+        </h1>
+        <header>
+          <figure className="profile-banner">

```

Comment:
Line 23:  Was this written with intent?  `<figure>`  I've never seen a `figure` tag before...